src/test/ui/did_you_mean/bad-assoc-ty.stderr

```diff
@@ -129,33 +129,18 @@ LL | fn foo<X: K<_, _>>(x: X) {}
    |             ^  ^ not allowed in type signatures
    |             |
    |             not allowed in type signatures
-   |
-help: use type parameters instead
-   |
-LL | fn foo<X: K<T, T>, T>(x: X) {}
-   |             ^  ^ ^^^
 
 error[E0121]: the type placeholder `_` is not allowed within types on item signatures
   --> $DIR/bad-assoc-ty.rs:52:34
    |
 LL | fn bar<F>(_: F) where F: Fn() -> _ {}
    |                                  ^ not allowed in type signatures
-   |
-help: use type parameters instead
-   |
-LL | fn bar<F, T>(_: F) where F: Fn() -> T {}
-   |         ^^^                         ^
```

Comment:
...if you look at these errors, there _are_ cases where we suggest a type parameter correctly. The big problem only happens on `static`s and `const`s are the enclosing item.